.github/workflows/mirrulations-CI-static-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: mirrulations-testing
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2 # Checkout the repository content to github runner.
+
+      - name: Setup Python Version
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9 # Install the python version needed
+
+      - name: Create Virtual Environment and installing mirrulations components as modules
+        run: |
+          python -m pip install --upgrade pip
+          python3 -m venv .venv
+          pwd
+          .venv/bin/pip install -e mirrulations-client
+          .venv/bin/pip install -e mirrulations-core
+          .venv/bin/pip install -e mirrulations-dashboard
+          .venv/bin/pip install -e mirrulations-mocks
+          .venv/bin/pip install -e mirrulations-work-generator
+          .venv/bin/pip install -e mirrulations-work-server
+
+      - name: Execute Makefile Unit Tests
+        run: make test
```

Comment:
Add a comment that this works because the `Makefile` modifies the path to use the `venv`